### PR TITLE
[translation] fix spelling according to cspell

### PR DESCRIPTION
**Pull request by kristapratico**
Resolves https://github.com/Azure/azure-sdk-for-python/issues/22684

### CODE REVIEW

**Review by seankane-msft — APPROVED** [review]

**Review by swathipil — APPROVED** [review]
lgtm!

File: sdk/translation/azure-ai-translation-document/tests/test_translation.py [hunk 1]
```diff
@@ -550,6 +550,6 @@ def test_translation_with_glossary(self, client):
 
         with open(document, "rb") as fd:
             translated = fd.readline()
-
+        # cSpell:ignore essai
```
Comment (danieljurek) [inline sdk/translation/azure-ai-translation-document/tests/test_translation.py:553]: I recommend using an override like this in the config so we don't couple our source code to the `cspell` tool.

```
    ...
    "overrides": [
      {
        "filename": "sdk/translation/azure-ai-translation-document/tests/*.py",
        "words": [
          "essai"
        ]
      },
      ...
    ]...

```

**Review by danieljurek — COMMENTED** [review]
Everything looks good. One small recommendation.

File: sdk/translation/azure-ai-translation-document/tests/test_translation.py (hunk 1 above)
Comment (kristapratico) [reply sdk/translation/azure-ai-translation-document/tests/test_translation.py:553]: Got it - thanks. I somehow missed that way of ignoring while reading the docs. Opened a PR to fix here: https://github.com/Azure/azure-sdk-for-python/pull/22865

**Pull request merged**